builder/azure/arm/artifact_test.go

```diff
@@ -16,7 +16,7 @@ func getFakeSasUrl(name string) string {
 func TestArtifactString(t *testing.T) {
 	template := CaptureTemplate{
 		Resources: []CaptureResources{
-			CaptureResources{
```

Comment:
FYI this is useful for compat with go 1.4. We've moved past this in packer but if you ever need to support 1.4 make sure to leave these types in.